PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -4195,6 +4197,30 @@ LRESULT FAR PASCAL FindReplaceDlg::comboEditProc(HWND hwnd, UINT message, WPARAM
 		delLeftWordInEdit(hwnd);
 		return 0;
 	}
+	else if (message == WM_SETFOCUS)
+	{
+		draftString[0] = '\0';
+	}

```

Comment:
It's tricky. I didn't find this solution 👍

Reply:
@donho Could you elaborate a bit what you didn't like about it?

I clear the draft string whenever this widget gets focus to ensure that the draft from another widget is not accidentally entered into the current one.

I still think that the feature has its merits, so if you could give me a hint what you don't like about this solution, I would like to find a better one.